src/wallet/wallet.cpp

```diff
@@ -3058,34 +3058,30 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, int nConfMine, int
 
 bool CWallet::SelectCoins(const CAmount& nTargetValue, set<pair<const CWalletTransactionBase*,unsigned int> >& setCoinsRet, CAmount& nValueRet,  bool& fOnlyCoinbaseCoinsRet, bool& fNeedCoinbaseCoinsRet, const CCoinControl* coinControl) const
 {
-    // If coinbase utxos can only be sent to zaddrs, exclude any coinbase utxos from coin selection.
-    bool fProtectCoinbase = Params().GetConsensus().fCoinbaseMustBeProtected;
-    bool fProtectCFCoinbase = false;
-
-    // CF exemption allowed only after hfCommunityFundHeight hardfork
-    if (!ForkManager::getInstance().canSendCommunityFundsToTransparentAddress(chainActive.Height()))
-        fProtectCFCoinbase = fProtectCoinbase;
+    // If coinbase utxos can only be sent to zaddrs, exclude any coinbase utxos from coin selection (based on current available forks).
+    const bool includeCoinBase = !ForkManager::getInstance().mustCoinBaseBeShielded(chainActive.Height() + 1);
+    const bool includeCommunityFund = ForkManager::getInstance().canSendCommunityFundsToTransparentAddress(chainActive.Height() + 1);
```

Comment:
About style consistency, I guess we should use `fIncludeCoinBase` and `fIncludeCommunityFund` also here.

Reply:
Yes, I renamed from `fMustShieldX` to `includeX`, forgetting about the prepended `f`; adapted accordingly with `fIncludeCoinBase` and `fIncludeCommunityFund`.